Bagdasaryan_Inga/Readme.md

```diff
@@ -0,0 +1,56 @@
+2. Follow the on-screen instructions to enter your Apple ID and password, then click Sign In. ![](photo.jpeg)
+
+> You can change your Apple ID name, password, or billing information at any time. You can also change the country or region.
+> 1. In the Podcasts app on your Mac, choose Account > View Apple ID to open the Media & Purchases options.
+> 2. Next to your account, click Manage. (You will need to log in).
+> 3. Do any of the following actions :
+>+ Change your Apple ID name or password. Click Edit at "appleid.apple.com".
+>+ Change your payment information. Click Manage Payments.
+>+ Change your billing address. Click Edit.
+>+ Change the country or region code. Click Change country or region.
+## Step 3 : Search and listen to podcasts
+1. In the Podcasts app on your device, click in the search box, then do one of the following:
++ To choose from the available categories, select a topic, such as Comedy or Sports.
++ To search all podcasts, enter a search term in the search box, then select one of the search results. Click "All Podcasts" on the right.
++ To search the media library, enter a search term in the search field, then select one of the search results. Click "Your media library".
+2. In the search results, do one of the following actions:
++ To play a show or episode, hover over the show or episode and click the Play button. See Listening to Podcasts.
++ To view show or episode details, under Shows, click the title link under the show. In the Episodes section, click the title link next to the episode.
++ To view channel details click a channel. (Subscription may be required for full access.)
++ To subscribe to a show or channel, select a show or channel, then click a subscription offer from the podcast creator. ![](podcasts.jpg)
+## Step 4: How to subscribe to a show or channel
+1. In the Podcasts app on your device, select the show or channel you want to subscribe to.When you subscribe to a show, you'll automatically follow it. See the show tracking section for details.
+2. Click the subscribe button. You may need to enter your Apple ID and password. 
+3. To confirm your subscription, select an available subscription offer from the podcast author (for example, annual or monthly).
+> You can view and change subscriptions purchased in the Podcasts app, the App Store, the Apple TV app, the iTunes Store, or Apple News.
+>1. In the Podcasts app on your device, choose Account > View Apple ID. The Media and Purchases options will open.
+>2. Click Manage next to Subscriptions.
+>3. Click Edit next to the subscription, then do one of the following:
+>+ To change your subscription options, select an option from the Options list and click Done.
+>+ To unsubscribe, click Unsubscribe, confirm the cancellation, and click Done. 
+
+![](podcasts_2.jpg)
+> **Share subscriptions in Apple Podcasts**
+> + When you subscribe to shows or channels, you can use Family Sharing to share your subscriptions among up to five family members.
```

Comment:
When you mention a feature that is a key point here - highlight its name in **Bold**